client/packages/inventory/src/Stocktake/context/stocktakeLineError.tsx

```diff
@@ -0,0 +1,63 @@
+import { PropsWithChildrenOnly, RecordWithId } from '@common/types';
```

Comment:
The context with errors seemed like the easier way to do it, at first I thought to just add a field to DraftStocktakeLine, but these errors would also need to be shown in main stocktake view and would need to update when they are fixed in batch view, so common context with errors seemed like the way to go